src/script/common/30-ssl.sh

```
@@ -5,58 +5,109 @@
+        if [[ -n "${SSL_CHAIN_FILE:-}" ]] && [[ -f "$SSL_CHAIN_FILE" ]]; then
+            if ! grep -q "SSLCertificateChainFile" /etc/apache2/sites-available/yii2.conf; then
+                sed -i "/SSLCertificateKeyFile/a\    SSLCertificateChainFile ${SSL_CHAIN_FILE}" /etc/apache2/sites-available/yii2.conf
+            else
+                sed -i "s|SSLCertificateChainFile .*|SSLCertificateChainFile ${SSL_CHAIN_FILE}|g" /etc/apache2/sites-available/yii2.conf
+            fi
+        fi
+    fi
+
+elif [[ "$SSL_AUTO_GENERATE" == "true" ]]; then
+    # DISABLE_OCSP stapling for self-signed certificates unless explicitly enabled
+    local DISABLE_OCSP="${DISABLE_OCSP_STAPLING:-true}"
```

Shellcheck (0.10.0)

[error] 'local' is only valid in functions.

(SC2168)